packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -620,17 +620,17 @@ export function isTransferConfirmed(tx: MergedTransaction) {
 }
 
 export function useRemainingTime(tx: MergedTransaction) {
```

Comment:
since this hook is only used for CCTP but has a pretty generic name, can you rename it to `useCctpRemainingTime` please?